sql/ddl_experimental.sql

```diff
@@ -37,3 +37,12 @@ CREATE OR REPLACE PROCEDURE timescaledb_experimental.copy_chunk(
     source_node NAME = NULL,
     destination_node NAME = NULL)
 AS '@MODULE_PATHNAME@', 'ts_copy_chunk_proc' LANGUAGE C;
+
+-- A copy_chunk or move_chunk procedure call involves multiple nodes and
+-- depending on the data size can take a long time. Failures are possible
+-- when this long running activity is ongoing. We need to be able to recover
+-- and cleanup such failed chunk copy/move activities and it's done via this
+-- procedure
+CREATE OR REPLACE PROCEDURE timescaledb_experimental.cleanup_copy_chunk_activity(
```

Comment:
I think we should be consistent about terminology, e.g., "activity" vs "operation" in this case. It makes more sense, I think, to name the function `cleanup_copy_chunk_operation` when the parameter name is `operation_id`. Otherwise it is more difficult to make the mental connection.

```suggestion
CREATE OR REPLACE PROCEDURE timescaledb_experimental.cleanup_copy_chunk_operation(
```

Reply:
Ok, done.